app/views/admin/editions/edit.html.erb

```diff
@@ -0,0 +1,38 @@
+<% content_for :page_title, "Edit publication"%>
+<% content_for :back_link, (render "govuk_publishing_components/components/back_link", {
+    href: admin_edition_path(@edition)
+  })
+%>
+<% content_for :error_summary, render('shared/error_summary', object: @edition.becomes(Edition), class_name: @edition.class.name.demodulize.underscore.humanize.downcase) %>
```

Comment:
Corr this is a mouthful! Perhaps there's a future scope for a user_facing_name method on Editions. Not in this PR of course as consistent with legacy view.

Reply:
I think this exists actually 🤔 

Let me find it and update it 👍